common/persistence/nosql/nosqlplugin/cassandra/gocql/session_test.go

```diff
@@ -52,6 +53,7 @@ func TestSessionEmitsMetricOnRefreshError(t *testing.T) {
 	metricsHandler.EXPECT().Counter(metrics.CassandraSessionRefreshFailures.Name()).Return(metrics.NoopCounterMetricFunc)
 
 	s.refresh()
+	controller.Finish()
```

Comment:
This doesn't seem necessary anymore. From the docs of [`gomoc.NewController()`](https://pkg.go.dev/github.com/golang/mock/gomock#NewController) - 
```
// New in go1.14+, if you are passing a *testing.T into this function you no
// longer need to call ctrl.Finish() in your test methods.
```

Reply:
good to know.